lib/models/command.js

```diff
@@ -9,15 +9,16 @@ var getCallerFile       = require('get-caller-file');
 var printableProperties = require('../utilities/printable-properties').command;
 var printCommand        = require('../utilities/print-command');
 var Promise             = require('../ext/promise');
-var union               = require('lodash/union');
-var uniq                = require('lodash/uniq');
-var uniqBy              = require('lodash/uniqBy');
-var map                 = require('lodash/map');
-var reject              = require('lodash/reject');
-var filter              = require('lodash/filter');
-var defaults            = require('lodash/defaults');
-var assign              = require('lodash/assign');
-var keys                = require('lodash/keys');
+var lodash = require('ember-cli-lodash-subset');
```

Comment:
should we assign to `_` instead of `lodash` and use the methods as e.g. `_.map()` instead of `map()` plus all the destructuring?